### DISCUSSION TITLE
纯小白求助

### DISCUSSION THREAD

**Discussion by MG190623** [Blinue/Magpie#412] [Q&A]
首先想知道怎么设置能让温度不要那么高，用的游戏本，显卡3060，但是开magpie之后玩一会就温度飙升到电脑提示温度过高，缩放模式用ACNet和anime4K都试过，都是温度飙升。感觉是不是我哪里没整好？设置缩放里有设置用3060。
然后也想知道怎么能设置那么多缩放模式，我只有初始的那几个，想要试下anime4K的其他模式来尝试降低功耗但是不知道咋设置

**Comment by Blinue** [comment]
确保开启了垂直同步。自定义缩放参见 https://github.com/Blinue/Magpie/wiki/%E8%87%AA%E5%AE%9A%E4%B9%89%E7%BC%A9%E6%94%BE%E9%85%8D%E7%BD%AE

**Reply by MG190623** [reply]
是开启了垂直同步，但是温度还是居高不下，下午改用FSR玩了，温度就还好。只要用anime4K或者ACNet风扇就狂转，温度一直升。

然后关于自定义缩放，实在不好意思，发的那个自己看了很多遍了，昨天研究了一下午也没整明白。实在是没搞明白怎么操作才会发帖子求助的。我看见了effect文件夹里面有那些自定义的hlsl文件，但如何把那个跟外面的json合起来实在没搞懂。

**Reply by hooke007** [reply]
那个文档里不是有现成的示例吗，直接抄不就完事了

**Reply by MG190623** [reply]
> 那个文档里不是有现成的示例吗，直接抄不就完事了

终于看明白了到示例里面原来有文件能直接下载，非常感谢。
